### PR TITLE
Exception handling service using Sentry

### LINKED ISSUE #774: Bug: Exceptional handling service
### Is your feature request related to a problem? Please describe.

At the moment we are not tracking errors from usage. Though the user count is low, if may be worth tracking for now. 

### Describe the solution you'd like

https://sentry.io/for/open-source/

or

https://www.bugsnag.com/open-source

### Describe alternatives you've considered

More tests.... :smile:

### Additional context

Does anyone else have options or opinions on this?

### PR DESCRIPTION
**Pull request by yg**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

Exceptional handling service using Sentry

## Related Tickets & Documents

Closes #774

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ import OneGraphApolloClient from "onegraph-apollo-client";
 import {ApolloProvider} from "react-apollo";
 import api from "./lib/apiGraphQL";
 import {getAppVersion} from "./lib/appVersion";
+import * as Sentry from "@sentry/react";
+
+Sentry.init({dsn: "https://33235628c0f14f559dc62e408e3b4299@o180224.ingest.sentry.io/5311329"});
```
Comment (yg) [inline src/index.js:14]: We can change this Public DSN before merging!

**Comment by yg** [comment]
Example of the know pagination error!

Screenshot

File: src/index.js [hunk 2]
```diff
@@ -9,6 +9,11 @@ import OneGraphApolloClient from "onegraph-apollo-client";
 import {ApolloProvider} from "react-apollo";
 import api from "./lib/apiGraphQL";
 import {getAppVersion} from "./lib/appVersion";
+import * as Sentry from "@sentry/react";
+import {isProd} from "./lib/isProd";
+
+isProd() &&
+  Sentry.init({dsn: "https://33235628c0f14f559dc62e408e3b4299@o180224.ingest.sentry.io/5311329"});
```
Comment (bdougie) [inline src/index.js:16]: @yg, should I create an open-sauced org or can you add me yours?

Comment (yg) [reply src/index.js:16]: @bdougie You create an open-sauced org and apply for a **sponsored account**! And add me to it!

If you want to try how it works I'll add you to my org!

Comment (bdougie) [reply src/index.js:16]: I can create the org and apply. I know folks at sentry and can follow up if needed

Comment (bdougie) [inline src/index.js:16]:
```suggestion
  Sentry.init({dsn: "https://c38b324e1edf48e08ce298a649417905@o416962.ingest.sentry.io/5314544"});
```
Adding the new sentry "sponsored" account

Comment (yg) [reply src/index.js:16]: Awesome 🎉

**Pull request closed without merging**